recipes-bsp/device-tree/files/uz7ev-evcc-quadcam-h/system-bsp.dtsi

```diff
@@ -1,7 +1,32 @@
-#include "system-conf.dtsi"
+#include "hdmi.dtsi"
```

Comment:
Is it needed to add the hdmi include here?
I thought the recipe was doing that with:

> 
> do_configure_append_uz7ev-evcc-hdmi () {
> 	if [ -e ${WORKDIR}/hdmi.dtsi ]; then
> 		cp ${WORKDIR}/hdmi.dtsi ${DT_FILES_PATH}/hdmi.dtsi
> 		echo '#include "hdmi.dtsi"' >> ${DT_FILES_PATH}/${BASE_DTS}.dts
> 	fi
> }

> do_configure_append_uz7ev-evcc-quadcam () {
> 	if [ -e ${WORKDIR}/fmc-quad.dtsi ]; then
> 		cp ${WORKDIR}/fmc-quad.dtsi ${DT_FILES_PATH}/fmc-quad.dtsi
> 		echo '#include "fmc-quad.dtsi"' >> ${DT_FILES_PATH}/${BASE_DTS}.dts
> 	fi
> }